Makefile

```diff
@@ -1,11 +1,11 @@
 
 install:
-	python2.7 -m virtualenv .
```

Comment:
I have `virtualenv` only installed into my Python 3.5 installation and wanted to be explicit about using Python 2.7 for this project. This refactor makes it work correctly either way.